easybuild/easyconfigs/p/picard/picard-2.20.6-Java-1.8.eb

```diff
@@ -0,0 +1,43 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+#
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   adelaide.edu.au/phoenix
+# Authors::	Robert Qiao <robert.qiao@adelaide.edu.au>, Exe Escobedo <exequiel.sepulvedaescobedo@adelaide.edu.au>
+# License::	MIT 
+#
+# 2.10.1:
+# Adam Huffman
+# The Francis Crick Institute
+# 2.18.11:
+# Jonas Demeulemeester
+# The Francis Crick Institute
+##
+
+easyblock = 'JAR'
+
+name = 'picard'
+version = '2.20.6'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'http://sourceforge.net/projects/%(name)s'
+description = """A set of tools (in Java) for working with next generation sequencing data in the BAM format."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/broadinstitute/%(name)s/releases/download/%(version)s']
+sources = ['%(name)s.jar']
```

Comment:
This will not work well, it will conflict with other versions. We must keep doing the same as the other picard easyconfigs.

Reply:
@Micket Thanks for your comment, now I see the purpose of the lines in the other easyconfigs.